examples/Huggingface_Transformers/Download_Transformer_models.py

```diff
@@ -0,0 +1,103 @@
+import transformers
+from pathlib import Path
+import os
+import json
+import torch
+from transformers import (AutoModelForSequenceClassification, AutoTokenizer, AutoModelForQuestionAnswering,
+ AutoModelForTokenClassification, AutoConfig)
+""" This function, save the checkpoint, config file along with tokenizer config and vocab files
+    of a transformer model of your choice.
+"""
+print('Transformers version',transformers.__version__)
+
+def transformers_model_dowloader(mode,pretrained_model_name,num_labels,do_lower_case):
+    print("Download model and tokenizer", pretrained_model_name)
+    #loading pre-trained model and tokenizer
+    if mode== "sequence_classification":
+        config = AutoConfig.from_pretrained(pretrained_model_name,num_labels=num_labels)
+        model = AutoModelForSequenceClassification.from_pretrained(pretrained_model_name, config=config)
+        tokenizer = AutoTokenizer.from_pretrained(pretrained_model_name,do_lower_case=do_lower_case)
+    elif mode== "question_answering":
+        model = AutoModelForQuestionAnswering.from_pretrained(pretrained_model_name)
+        tokenizer = AutoTokenizer.from_pretrained(pretrained_model_name,do_lower_case=do_lower_case)
+    elif mode== "token_classification":
+        config = AutoConfig.from_pretrained(pretrained_model_name,num_labels=num_labels)
+        model = AutoModelForTokenClassification.from_pretrained(pretrained_model_name)
+        tokenizer = AutoTokenizer.from_pretrained(pretrained_model_name,do_lower_case=do_lower_case)
+
```

Comment:
Should we add a else warning here if the mode is not supported?

Reply:
Thanks for your suggestion, the warning is added now.